### ISSUE TITLE
Release 8.1.1 on March 1, 2021

### ISSUE THREAD

**Issue by radarhere** [python-pillow/Pillow#5295]
* [x] Make necessary changes in master branch.
* [x] Update CHANGES.rst.
* [x] Check out release branch e.g.:
```
git checkout -t remotes/origin/8.1.x
```
* [x] Cherry pick individual commits from master branch to release branch e.g. 8.1.x, then git push.
* [x] Check GitHub Actions and AppVeyor to confirm passing tests in release branch e.g. 8.1.x.
* [x] In compliance with PEP 440, update version identifier in src/PIL/_version.py
* [x] Run pre-release check via make release-test.
* [x] Create tag for release e.g.:
```
git tag 8.1.1
git push
git push --tags
```
* [x] Create and check source distribution:
```
make sdist
twine check dist/*
```
* [x] Create binary distributions
* [x] Check and upload all binaries and source distributions e.g.:
```
twine check dist/*
twine upload dist/Pillow-8.1.1*
```
* [x] Publish the release on GitHub

**Comment by radarhere** [comment]
@cgohlke could we have Windows binaries for 8.1.1?

**Comment by cgohlke** [comment]
[Here](https://www.lfd.uci.edu/~gohlke/download/Pillow-8.1.1-win-dist.zip) you go.

**Comment by radarhere** [comment]
Thanks.

GitHub Actions doesn't seem to want to build the tagged release for pillow-wheels - https://github.com/python-pillow/pillow-wheels/actions. So I created the GitHub Actions builds using my fork (which also means that we have an M1 wheel for the first time). I also created the aarch64 builds in my Travis account.

And we're done - https://twitter.com/PythonPillow/status/1366550467845623808?s=20
